src/Quartz/Simpl/DefaultObjectSerializer.cs

```diff
@@ -12,6 +16,35 @@ namespace Quartz.Simpl
     /// <author>Marko Lahma</author>
     public class DefaultObjectSerializer : IObjectSerializer
```

Comment:
would separate JsonObjectSerializer and BinaryObjectSerializer now be the correct separation? We could make it mandatory for user to define in configuration.